CHANGES.md

```diff
@@ -67,6 +67,7 @@
     [Ensuring Python Type Safety](https://beam.apache.org/documentation/sdks/python-type-safety/)
     and an upcoming
     [blog post](https://beam.apache.org/blog/python/typing/2020/03/06/python-typing.html).
+* --direct_num_workers=0 is supported for FnApi. It will set the number of threads/subprocesses to number of cores of the machine executing the pipeline([BEAM-9443](https://issues.apache.org/jira/browse/BEAM-9443)).
```

Comment:
It is added to 2.21.0, please let me know if the branch is already cut, I can move it to 2.22.0 if needed.

Reply:
The branch is cut, please move to 2.22.0